### DISCUSSION TITLE
How to add cards to a deck that belongs to another deck?

### DISCUSSION THREAD

**Discussion by RubickWilde** [ObsidianToAnki/Obsidian_to_Anki#226] [Q&A]
![image](https://user-images.githubusercontent.com/41550137/110106497-6c539600-7db2-11eb-9726-0df5ee6beb34.png)

What should I wright in TARGET DECK to add cards to specific deck that is in another deck? If I wright just the name of the deck, a new deck will be created.

**Comment by chanithecat** [comment]
TARGET DECK: Test::Test1
